itou/www/stats/views.py

```diff
@@ -521,6 +529,32 @@ def stats_ph_beneficiaries(request):
     )
 
 
+@check_request(utils.can_view_stats_ph)
+def stats_ph_raw(request):
+    return render_stats_ph(
+        request=request,
+        page_title="Export complet de vos données",
+        with_department_param=False,
+        with_department_name=False,
```

Comment:
Ça ne change rien au fonctionnement mais dans le cas actuel on aura préférence à ne pas passer `with_department_name` quand `with_department_param=False`, si il n'y a pas de paramètre alors pas de raison d'avoir son nom ou pas son nom ;), ce qui évite une informations peu utile (voir trompeuse) quand un humain passe sur le code.
```suggestion
        with_department_param=False,
```